playbook/app/pb_kits/playbook/pb_avatar/avatar.rb

```diff
@@ -25,6 +25,14 @@ def online_status_props
         { status: status, classname: "size_#{size}" }
       end
 
+      def alt_text
+        if image_alt.present?
+          image_alt
+        else
+          name
+        end
```

Comment:
```suggestion
        image_alt.present? ? image_alt : name
```
Ruby ternary is similar syntax to typescript